src/Core/Settings/GlobalSettings.cs

```diff
@@ -57,6 +57,7 @@ public virtual string LicenseDirectory
     public virtual ConnectionStringSettings Events { get; set; } = new ConnectionStringSettings();
     public virtual IConnectionStringSettings Redis { get; set; } = new ConnectionStringSettings();
     public virtual NotificationsSettings Notifications { get; set; } = new NotificationsSettings();
+    public virtual GrantsSettings Grants { get; set; } = new GrantsSettings();
```

Comment:
⚠️ Do we want to overdesign a little here? Could reference this as some kind of "entity caching" and have sub-objects. Alternatively, this could just go in the IdentityServer configuration block.